raiden/storage/sqlite.py

```diff
@@ -417,6 +430,33 @@ def get_statechanges_by_identifier(self, from_identifier, to_identifier):
         result = [entry[0] for entry in cursor]
         return result
 
+    def _write_matrix_room_ids_for_address(self, room_id_data):
+        with self.write_lock, self.conn:
+            self.conn.execute("INSERT INTO room_ids_and_aliases VALUES(?, ?, ?, ?)", room_id_data)
+
+    def _get_matrix_room_ids_aliases_for_address(self, address_identifier):
+        cursor = self.conn.cursor()
+        cursor.execute(
+            "SELECT room_ids_and_aliases, address FROM room_ids_and_aliases "
+            "WHERE address_identifier = ?",
+            (address_identifier,),
+        )
+        rows = cursor.fetchall()
+        room_ids_to_aliases, stored_address = rows[0][0], rows[0][1]
+        return room_ids_to_aliases, stored_address
+
+    def _write_matrix_user_ids_for_address(self, user_id_data):
+        with self.write_lock, self.conn:
+            self.conn.execute("INSERT OR REPLACE INTO user_ids VALUES(?, ?,?, ?) ", user_id_data)
```

Comment:
nitpick: space after the second comma

Reply:
Also a question: What is the point of storing "user_ids" for a certain node?

Reply:
We can't send messages directly to a node but only to one (of potentially many) user_ids this node has used. Therefore we currently query the user_directory on the server for all user_ids for a given address, which is slower and more error prone than reading from disk and only falling back to the server if we couldn't find an appropriate user.